addons/safemode/functions/fnc_unlockSafety.sqf

```diff
@@ -17,7 +18,7 @@
  * Public: No
  */
 
-params ["_unit", "_weapon", "_muzzle"];
+params ["_unit", "_weapon", "_muzzle", "_hint"];
```

Comment:
https://github.com/acemod/ACE3/blob/master/addons/weaponselect/functions/fnc_selectWeaponMode.sqf#L27-L30
calls this with `[_unit, _weapon, _weapon] call EFUNC(safemode,unlockSafety);`

```suggestion
params ["_unit", "_weapon", "_muzzle", ["_hint", true, [true]]];
```